### ISSUE TITLE
Missing gossip on screen

### ISSUE THREAD

**Issue by legdrop** [CQUI-Org/cqui#517]
Good afternoon, firstly, thank you very much for such a wonderful modification, which I use with the acquisition of the game.
But there is a problem - gossip on the screen about AI does not appear at all, just notifications about my studied technologies of culture and science.
How can I solve this problem? Or tell me, which file or string I should cut to get rid of the gossip removal function.
I'm playing the Russian version of the game.
Thank you very much!

**Comment by chaorace** [comment]
This may be an issue with the heuristic used for intercepting gossips. I did my best to render it language neutral, but there's only so much I can achieve as an English-only speaker.

Hey @e1ectron, does this issue affect you as well?

**Comment by e1ectron** [comment]
Sorry, I can't check it now and in the near future (don't have access to actual game version).

**Comment by e1ectron** [comment]
Hey, @chaorace. Test it on a 1.0.4 (1.0.0.129) Mac version on Russian and English. 
For a hour, I haven't seen any gossips. So problem is not the language. 

May be platform? @legdrop, you play on Win or Mac?

**Comment by legdrop** [comment]
Windows 10, 64 bit.

**Comment by bolbass** [comment]
I think @legdrop wants to read popup gossips that appears on vanilla on center of the screen. If so you need to remove line 35 in UI/Panels/StatusMessagePanel.lua that was added on this commit f04d0b5e0ff387a80420ccb0009865df134fc524

**Comment by legdrop** [comment]
@bolbass
Yes, you understood everything correctly, thank you all very much!
I do not have the option now to run the game, but please check, I selected the correct code to delete?
![1](https://cloud.githubusercontent.com/assets/20305458/26274308/e5314510-3d4f-11e7-99fb-f3d1d2b18873.png)

**Comment by bolbass** [comment]
Don't delete it :)
The commit I mentioned above was already fixed in CQUI so my last comment is wrong.
The last commit that affects gossips is c8963130252a8cb90546423564eb6c22911e64ea
The result of this commit is that you can choose gossips you want to see or disable on CQUI settings that are above the minimap. Have you tried changing it?

**Comment by legdrop** [comment]
Thanks a lot, it works!

**Comment by chaorace** [comment]
Okay... so did we ever figure out if this issue was caused by CQUI not correctly showing gossips?

**Comment by bolbass** [comment]
Yes, there is at least one type of gossips that is missing on gossips settings screen (a gossip when other civs enter new era for example).

**Comment by bolbass** [comment]
This type of gossips is under "City>>X has counquered Y" settings. We should fix it and can close this issue as soon as it's enough to unchek "Trim gossip messages" to see all the gossips.
